test/e2e/step_definitions/steps.js

```diff
@@ -0,0 +1,20 @@
+const {client} = require('nightwatch-cucumber');
+const {defineSupportCode} = require('cucumber');
+
+defineSupportCode(({Given, Then, setDefaultTimeout}) => {
+    setDefaultTimeout(30 * 1000);
```

Comment:
Timeouts configuration could be extracted to a separate file. More details: https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/timeouts.md